cookbook/prompt_management_openai_functions.ipynb

```diff
@@ -208,36 +208,23 @@
       ]
```

Comment:
<p>I'd suggest to only move the output to a md block, we still need to call summarize story to produce an actual trace in this notebook</p><p> </p>

<pre class="ql-syntax" spellcheck="false">summary = summarize_story(STORY) 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>